bin/install_openstackbmc.sh

```diff
@@ -45,12 +45,25 @@ export OS_TENANT_NAME=$os_tenant
 export OS_PASSWORD=$os_password
 export OS_AUTH_URL=$os_auth_url
 export OS_PROJECT_NAME=$os_project
-export OS_USER_DOMAIN_ID=$os__user_domain
-export OS_PROJECT_DOMAIN_ID=$os__project_domain
+export OS_USER_DOMAIN_ID=$os_user_domain
+export OS_PROJECT_DOMAIN_ID=$os_project_domain
+
+# Following is required so that openstack CLIs work with DOMAIN_NAME credentials
+export OS_USER_DOMAIN_NAME=$os_user_domain
+export OS_PROJECT_DOMAIN_NAME=$os_project_domain
+
 # v3 env vars mess up v2 auth
 [ -z $OS_PROJECT_NAME ] && unset OS_PROJECT_NAME
 [ -z $OS_USER_DOMAIN_ID ] && unset OS_USER_DOMAIN_ID
 [ -z $OS_PROJECT_DOMAIN_ID ] && unset OS_PROJECT_DOMAIN_ID
+[ -z $OS_USER_DOMAIN_NAME ] && unset OS_USER_DOMAIN_NAME
+[ -z $OS_PROJECT_DOMAIN_NAME ] && unset OS_PROJECT_DOMAIN_NAME
+# Check if v3 DOMAIN_ID credential works
+neutron net-list
+if [ "$?" != "0" ]; then
```

Comment:
I would mildly prefer to check the return code directly with:

if neutron net-list; then
...
fi

Checking $? is not set -e compatible if we ever decide to set that in this script.